.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+        args: [--unsafe]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+  - repo: https://github.com/pycqa/isort
+    rev: 5.13.2  # Use the same version as in pyproject.toml
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--filter-files"]
```

Comment:
Can you please configure the profile in `pyproject.toml` so that when I run `isort` directly from CLI I get the same results. Or it won't have effect on `pre-commit`?

```toml
[tool.isort]
profile = "black"
```

https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#profile

Reply:
Done! d929251

Reply:
Silly me forgot to save `pyproject.toml`, fixed in 3e7d13d.